python/pyspark/sql/column.py

```diff
@@ -224,7 +224,39 @@ def __init__(self, jc):
        https://spark.apache.org/docs/latest/sql-programming-guide.html#nan-semantics
     .. versionadded:: 2.3.0
     """
+    _isNotDistinctFrom_doc = _eqNullSafe_doc.replace("eqNullSafe", "isNotDistinctFrom")
```

Comment:
![2017-05-02 12 41 27](https://cloud.githubusercontent.com/assets/6477701/25603393/3b0db43c-2f36-11e7-932c-135b7e7d4ca3.png)

This is the same with `eqNullSafe` but only the word `eqNullSafe` was replaced to `isNotDistinctFrom`.